src/typography/expandable-text.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ExpandableText = ({ title, children }) => (
+  <details>
+    <summary><span className="summary">{title}</span></summary>
+    <div className="panel panel-border-narrow">
+      {children}
+    </div>
+  </details>
+);
+
+ExpandableText.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired
```

Comment:
I think I missed this on #22 but we should probably also pass in an optional [`open` prop as per the spec](https://dev.w3.org/html5/spec-preview/the-details-element.html)